controllers/snapshot/snapshot_adapter.go

```diff
@@ -351,6 +370,23 @@ func (a *Adapter) createIntegrationPipelineRun(application *hasv1alpha1.Applicat
 		WithIntegrationLabels(integrationTestScenario).
 		WithApplicationAndComponent(a.application, a.component).
 		AsPipelineRun()
+	//Copy PipelineRun PAC annotations/labels from applicaitonSnapshot to integration test PipelineRuns
+	if applicationSnapshot.Annotations != nil {
+		pipelineRun.ObjectMeta.Annotations = make(map[string]string)
+		for key, value := range applicationSnapshot.Annotations {
+			if strings.Contains(key, "pipelinesascode.tekton.dev") {
+				pipelineRun.ObjectMeta.Annotations[key] = value
+			}
+		}
+	}
+	if applicationSnapshot.Labels != nil {
+		pipelineRun.ObjectMeta.Labels = make(map[string]string)
```

Comment:
This line will erase all labels that were already set on the `pipelineRun` object. I'd say we want to do the `make` call only if the `pipelineRun.ObjectMeta.Labels` are `nil`.

Reply:
Yes. It need to be removed.

Reply:
Yes, looks good now, thanks!